pyamg/util/linalg.py

```diff
@@ -6,7 +6,7 @@
 import scipy as sp
 import scipy.sparse as sparse
 from scipy.linalg.lapack import get_lapack_funcs
-from scipy.linalg import calc_lwork
+from scipy.linalg.lapack import _compute_lwork
```

Comment:
Is there a public API version of this? If not, I worry that you are going to run into the same issue of using a function that is removed in the future...

Reply:
Here?
https://github.com/pyamg/pyamg/commit/40d1b26294f6b516699c333ea4ffc16f1d2dfd32

It's in `master` and `3.3.2` now.

Reply:
My concern is that the name `_compute_lwork` indicates non-public API in scipy.